index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>My First Quiz</title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+
+<body>
+<!-- first card box -->
+  <article class="firstcard">
+    <header class="top"><h1>Dull Playing Cards</h1></header>
```

Comment:
It isn't a requirement or anything, but typically when you've got nested tags like you space it out to make it a little more readable: 
```    <header class="top">
      <h1>Dull Playing Cards</h1>
    </header>```